packages/opam-custom-install/opam-custom-install.0.2/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+maintainer: "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+synopsis: "An opam plugin to install a package using a custom command"
+description: """
+Provides the `opam custom-install` command, which allows to wrap a custom install command, and make opam register it as the installation of a given package. This is a prototype provided for the moment as a plugin, but might get integrated into opam if useful.
+"""
+tags: ["org:ocamlpro" "org:opam"]
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+depends: [
+  "dune" {>= "1.5"}
+  "opam-client" {>= "2.1.2"}
+]
+homepage: "https://gitlab.ocamlpro.com/louis/opam-custom-install"
+bug-reports: "https://gitlab.ocamlpro.com/louis/opam-custom-install/-/issues"
+dev-repo: "git+https://gitlab.ocamlpro.com/louis/opam-custom-install.git"
+build: ["dune" "build" "-p" name "-j" jobs]
+flags: plugin
+authors: "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+url {
+  src:
+    "https://gitlab.ocamlpro.com/louis/opam-custom-install/-/archive/0.1/opam-custom-install-0.2.tar.gz"
```

Comment:
Also having both `0.1` and `0.2` in the url doesn’t seem right. Are you sure this is the right archive?

Reply:
Thanks!! :sweat_smile: this was driving me crazy

Reply:
so ok actually github accepts https://gitlab.ocamlpro.com/louis/opam-custom-install/-/archive/0.1/ANYTHING.tar.gz" :man_shrugging: thanks for being helpful...

Reply:
s/github/gitlab/